### ISSUE TITLE
vimium is not working on some websites

### ISSUE THREAD

**Issue by cserb** [philc/vimium#2756]
I can't figure out what might cause the problem but on some websites are not working with vimium.
One website that never works is the addons page of mozilla: https://addons.mozilla.org/en-US/firefox/  (I didn't add any exception. Actually I removed all just to make sure)

I'm on Arch Linux with FF 56.0.2 (64-bit)

**Comment by mrmr1993** [comment]
Firefox deliberately blocks us from working there, so we e.g can't block you from uninstalling the extension or install others covertly. Notably, Chrome blocks Vimium on their web store too. There's nothing we can do here.

**Comment by cserb** [comment]
Thanks for the answer!
One more question: Why is `x = removeTab` not working on duckduckgo or google?

**Comment by smblott-github** [comment]
@cserb... Cannot reproduce.  `x` working fine on these sites for me.

**Comment by cserb** [comment]
@smblott-github true, I found out that by coincidence the focus was not within the website for both. Which leads me to the next question. (sorry)
Coming from VimFx I was used to press `ESC` to unfocus everything and than `x` or whatever. It seems that now even focusing the address bar is not affected by `ESC`. Is this a limitation of the plugin (57+), expected behaviour or just not implemented yet?

**Comment by smblott-github** [comment]
Once the focus is outside of the page, Vimium does not see keyboard events, and so cannot respond to them.  There is nothing we can do about this.

**Comment by snsnsjsn** [comment]
vimium ff now work on mozilla.org like this link https://support.？/zh-CN/kb/键盘快捷键#firefox:win10:fx68 
but chrome vimium c can work 
so how to setting firefox to enable vimium this page?

**Comment by gdh1995** [comment]
@snsnsjsn your URL above is broken, please update it - you may use "``` ` ```" to wrap the url, and then it will not be escaped by GitHub.

**Comment by snsnsjsn** [comment]
k

> @snsnsjsn your URL above is broken, and maybe it is "[https://support.mozilla.org/zh-CN/kb/键盘快捷键#firefox:win10:fx68](https://support.mozilla.org/zh-CN/kb/%E9%94%AE%E7%9B%98%E5%BF%AB%E6%8D%B7%E9%94%AE#firefox:win10:fx68)".
> 
> Firefox has some priviledged "domains" which don't allow extension to run, and "*.mozilla.org" is just one, so both Vimium FF and Vimium C (Firefox version) can not work on it. However this domain is not protected on Chrome - this is why Vimium C (Chrome version) can work.
> 
> Chrome also has some similar domains like "https://chrome.google.com/webstore/***", on which no extensions can run.

i see ,  chrome vimium can run on  "*.mozilla.org"
and firefox vimium can run on https://chrome.google.com/webstore/category/extensions

thanks

**Comment by Atemu** [comment]
In Firefox you can disable this behaviour by removing the websites from `extensions.webextensions.restrictedDomains`, adding the boolean `privacy.resistFingerprinting.block_mozAddonManager` and setting it to `true`.

[Source](https://www.reddit.com/r/firefox/comments/e2o7qm/why_does_this_extension_not_work_on_mozilla/f908y1u/)

**Comment by UtmostCreator** [comment]
Hi, could you please have a look at this extension:
https://github.com/harshayburadkar/clut-chrome-extension
https://chrome.google.com/webstore/detail/clut-cycle-last-used-tabs/cobieddmkhhnbeldhncnfcgcaccmehgn/
http://www.harshay-buradkar.com/clut_update6.html

This extension at least works on every page with the following commands:
```
Quick switch (default: Alt + W)
Normal switch (default: Alt + S)
Normal switch (default: Alt + Shift + S)
```

Could you please have a look at how the same thing could be done in your extension?
At least, I would like to be able to switch between tabs (current and previous tab) as the `CLUT` extension does. 

Thank you!

**Comment by gdh1995** [comment]
@UtmostCreator Your issue is another problem, and will be solved by #3785 . 

You may also use my customized version of Vimium, `Vimium C` (https://github.com/gdh1995/vimium-c), and it support such global shortcuts. You may see https://github.com/gdh1995/vimium-c/wiki/Trigger-commands-in-an-input-box#user-content-shortcuts .

**Comment by fingertap** [comment]
> In Firefox you can disable this behaviour by removing the websites from `extensions.webextensions.restrictedDomains`, adding the boolean `privacy.resistFingerprinting.block_mozAddonManager` and setting it to `true`.
> 
> [Source](https://www.reddit.com/r/firefox/comments/e2o7qm/why_does_this_extension_not_work_on_mozilla/f908y1u/)

Hi. Where can I set these properties?
